src/APIResponder.jl

```diff
@@ -128,22 +136,50 @@ end
 function process(apispecs::Array, addr::AbstractString=get(ENV,"JBAPI_QUEUE",""); log_level=INFO, bind::Bool=false, nid::AbstractString=get(ENV,"JBAPI_CID",""))
     setup_logging()
     Logging.debug("queue is at $addr")
-    api = APIResponder(addr, Context(), bind, nid)
+    api = create_responder(apispecs, addr, bind, nid)
+
+    Logging.debug("processing...")
+    process(api)
+
+end
+
+function process_async(apispecs::Array, addr::AbstractString=get(ENV,"JBAPI_QUEUE",""); log_level=INFO, bind::Bool=false, nid::AbstractString=get(ENV,"JBAPI_CID",""))
+    setup_logging()
+    Logging.debug("queue is at $addr")
+    api = create_responder(apispecs, addr, bind, nid)
 
+    Logging.debug("processing...")
+    @async process(api)
+    return api
+end
+
+function create_responder(apispecs::Array{Tuple, 1}, addr, bind, nid)
```

Comment:
signature should probably be `create_responder(apispecs::Array, addr, bind, nid)`